source/simulator/solver/stokes_matrix_free_global_coarsening.cc

```diff
@@ -1574,7 +1574,7 @@ namespace aspect
 
             if (this->get_boundary_velocity_manager().get_tangential_boundary_velocity_indicators().size() > 0)
               {
-                Assert(false, dealii::StandardExceptions::ExcNotImplemented("This is not tested."));
+                // Assert(false, dealii::StandardExceptions::ExcNotImplemented("This is not tested."));
```

Comment:
If you remove the assert, just remove it. The current state of the code base should represent the current state, not some previously present assertion -- that is just bound to confuse readers. If you think the code is now well tested, let's just remove the assertion. If you *don't* think that the code is well tested, then of course we should not remove the assertion.